### DISCUSSION TITLE
@import in CSS file is not work

### DISCUSSION THREAD

**Discussion by wangcongyi** [web-infra-dev/rsbuild#3945] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a1b23ff6-6107-4aa7-b244-18dc8bfb5650)

写在 css 文件中的 @import 无效果，放在其他文件 另外引入就可以了。

从 craco 里迁移过来使用 rspack 的

rsbuild.config.js
![image](https://github.com/user-attachments/assets/bedd008b-2e69-4b7c-ba5e-8e30fe55534a)

**Comment by wangcongyi** [comment]
![image](https://github.com/user-attachments/assets/56e5482d-871d-4ca1-8ccd-dd4e1a76236c)
有时候 tailwind css 的 media 也丢失样式，

**Comment by 9aoy** [comment]
Please provide a reproduction repository.

**Comment by wangcongyi** [comment]
@9aoy   https://github.com/wangcongyi/demo-rsbuild

使用 rsbuild  会把  tailwind css    lg: xxxx  这种样式没有编译，   用 CRA 则是可以的
![image](https://github.com/user-attachments/assets/aedc3062-6fde-4da4-a183-bb12dcd4b64f)

**Reply by wangcongyi** [reply]
![image](https://github.com/user-attachments/assets/22f61eb2-5c5e-4d03-a7ad-12c3d93fe405)

**Reply by 9aoy** [reply, marked as answer]
You need to register the tailwindcss PostCSS plugin first. 
https://rsbuild.dev/guide/basic/tailwindcss#configuring-postcss

**Reply by wangcongyi** [reply]
谢谢 已经好了  从10多分钟  变成了 18s   太感谢了
